exampleSite/content/_index/header.md

```diff
@@ -0,0 +1,11 @@
++++
+date = "2018-07-13"
+fragment = "header"
+weight = 500
+
+background = "white"
+title = "Section Header Fragment"
+subtitle = "Even linking is possible. This fragment can be used for related sections so linking is easier."
+#align = "left" # Default center, available values: left, center, right, with bootstrap breakpoint variables (default: xs, sm, md, lg, xl)
```

Comment:
"exposing" the breakpoint variables seems imperfect. Yes it works, but is that the configuration interface we want to expose? Especially through "align"?

Reply:
Since we're doing `text-%s` it's possible to use `text-md-center` for example since Bootstrap allows it. I can remove the documentation but seemed better to have it.

Reply:
Mhh I would leave it out for now. Yes we can do that, but I would discuss that separately. Especially as we don't do it with any other alignment yet. It would be a break of consistency. Let's open a separate issue for that and discuss.